### PR TITLE
Remove all non-Brave blessed extensions

### PR DESCRIPTION
**Pull request by pes10k**
This is part of a larger project to remove this (extension-whitelist) dependency all together

### CODE REVIEW

File: data/whitelist.json [hunk 1]
```diff
@@ -48,42 +20,6 @@
...
-            "id": "dbepggeogbaibhgnhhndojpepiihcmeb",
-            "description": "Vimium"
-        },
-        {
-            "id": "kbmfpngjjgdllneeigpgjifpgocmfgmb",
-            "description": "Reddit Enhancement Suite"
-        },
-        {
-            "id": "ahfgeienlihckogmohjhadlkjgocpleb",
-            "description": "Web Store"
```
Comment (fmarier) [inline data/whitelist.json:85]: That one is not an extension (https://chrome.google.com/webstore/detail/line-share/ahfgeienlihckogmohjhadlkjgocpleb). I'm not sure what it is, but we should probably leave it in until we can identify it.

**Review by fmarier — CHANGES_REQUESTED** [review]

**Comment by fmarier** [comment]
https://github.com/brave/brave-browser/wiki/Extension-whitelisting-process will need to be updated once this is merged.

File: data/whitelist.json (hunk 1 above)
Comment (pes10k) [reply data/whitelist.json:85]: sounds good, done and done

File: data/whitelist.json [hunk 2]
```diff
@@ -1,33 +1,5 @@
...
-        {
-            "id": "khgocmkkpikpnmmkgmdnfckapcdkgfaf",
-            "description": "1PasswordX Beta"
-        },
-        {
-            "id": "ajopnjidmegmdimjlfnijceegpefgped",
-            "description": "BetterTTV"
-        },
-        {
-            "id": "mfehgcgbbipciphmccgaenjidiccnmng",
```
Comment (fmarier) [inline data/whitelist.json:20]: That's also not a real extension: https://chrome.google.com/webstore/detail/line-share/mfehgcgbbipciphmccgaenjidiccnmng

Probably some kind of Chromium component that gets installed automatically under some circumstances.

File: data/whitelist.json [hunk 3]
```diff
@@ -1,33 +1,5 @@
...
-            "description": "Cloud Print"
-        },
-        {
-            "id": "kmendfapggjehodndflmmgagdbamhnfd",
-            "description": "CryptoTokenExtension"
```
Comment (fmarier) [inline data/whitelist.json:25]: That's probably some kind of Brave thing. Also not on the extension store.

**Review by fmarier — CHANGES_REQUESTED** [review]
Two more we should keep. I think all of the other ones are regular third-party extensions.

File: data/whitelist.json (hunk 3 above)
Comment (bsclifton) [reply data/whitelist.json:25]: I think this is something we used in Muon to have U2F support

File: data/whitelist.json (hunk 2 above)
Comment (pes10k) [reply data/whitelist.json:20]: okay ive added this back in

File: data/whitelist.json (hunk 3 above)
Comment (pes10k) [reply data/whitelist.json:25]: Sounds good i'll keep it deleted

**Review by fmarier — APPROVED** [review]

**Pull request merged**